lib/Data/Validate/Sanctions.pm

```diff
@@ -16,7 +16,7 @@ use Scalar::Util qw(blessed);
 use Date::Utility;
 use List::Util qw(any uniq);
 
-our $VERSION = '0.11';
+our $VERSION = '0.12';
```

Comment:
This should be updated only upon `dzil release` (and the corresponding Changes version bumped automatically)